xml/dm_registers.xml

```diff
@@ -17,7 +17,14 @@
         time.  Eg. in a multi-hart system some might temporarily be powered
         down, or a system might support hot-swapping harts.
 
-        <field name="0" bits="31:21" access="R" reset="0" />
+        <field name="0" bits="31:23" access="R" reset="0" />
+        <field name="impebreak" bits="22" access="R" reset="Preset">
+            If 1, then there is an implicit {\tt ebreak} instruction that will
+            be executed if execution runs off the end of the Program Buffer.
```

Comment:
can we make this a bit more clear... If program buffer word [\Fprogbufsize -1] is executed. To make it clear what is "the end" of program buffer.

Reply:
I started down that route, but gave up when I couldn't think of a good phrasing for:
After the last instruction in the Program Buffer is executed, and does not change the PC in any way besides normal incrementing...

Suggestions?

Reply:
Hmm, how about just saying there is an implicit {\tt ebreak} at the (nonexistent) ProgBuf location ProgBuf[\Fprogbufsize]. That's really what this is saying.